modules/rest-api/pages/rest-xdcr-adv-settings.adoc

```diff
@@ -320,68 +382,118 @@ It cannot be set when an individual replication is created, or when its settings
 | Default: 4.
 The maximum number of threads used per node, to support XDCR.
 A greater number of threads increases parallelism, and may thereby produce enhanced XDCR performance.
+
 This setting is only returned when global settings are retrieved; and can only be set when global settings are set.
 It cannot be set when an individual replication is created, or when its settings are modified.
 
+| `hlvPruningWindowSec`
+| Integer
+| Used in conflict-resolution. Developer Preview only.
+
+This setting can be established and retrieved either for an individual replication or globally.
+
+| `jsFunctionTimeoutMs`
+| Integer
+| Used in conflict-resolution. Developer Preview only.
```

Comment:
Used in *custom* conflict-resolution only

Reply:
Users can specify merge javascript functions to use when conflicts arise, to resolve the conflicts into a single document. The javascript merge function must complete within this timeout period in millisecond, otherwise the merge will be aborted

Reply:
I have written: Used in _custom conflict-resolution_, and available in Developer Preview Mode only.
Specifies the number of milliseconds that must elapse before the timeout of a JavaScript conflict-resolution function.